doc-content/jbpm-docs/src/main/asciidoc/BAM/DashbuilderRuntime-section.adoc

```diff
@@ -16,36 +16,156 @@ After login, users may upload dashboard files exported from {CENTRAL}
+
+... Click *Finish*.
+
+--
+
+== Importing Dashbuilder data
+
+You can import Dashbuilder data to {CENTRAL} from a ZIP file if the archive is structured in the same way as the following example:
+
+[source]
+----
+dashbuilder/datasets/definitions/dataset-example1.dset
+dashbuilder/datasets/definitions/dataset-example2.dset
+dashbuilder/datasets/readme.md
+dashbuilder/perspectives/page1/perspective_layout
+dashbuilder/perspectives/page1/perspective_layout.plugin
+dashbuilder/perspectives/page2/perspective_layout
+dashbuilder/perspectives/page2/perspective_layout.plugin
+dashbuilder/perspectives/readme.md
+dashbuilder/navigation/navigation/navtree.json
+dashbuilder/navigation/readme.md
+VERSION
+----
+
+.Procedure
+. In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Dashbuilder Data Transfer*.
++
+WARNING: You should only import Dashbuilder data to a clean installation of {PRODUCT} in order to avoid overwriting data on an existing system.
+
+. On the *Dashbuilder Data Transfer* page, click the *Choose File* icon.
+
+. Navigate to the ZIP file you want to import and select the file.
+
+. Click the *Upload* icon.
+
+. Click *Import*.
+
+== Deploying dashboards from {CENTRAL} on dashbuilder runtime
+
+You can automatically deploy the dashboards from {CENTRAl} on dashbuilder runtime. {CENTRAl} is linked to dashbuilder runtime with the help of gradual export feature.
+
+.Prerequisites
+
+* Dasbuilder runtime is configured on the system.
+* You have set the `dashbuilder.runtime.multi` system property to `true` in the `standalone.xml` file.
+* You have set the value of `dashbuilder.runtime.location` system property to the Dashbuilder Runtime URL.
++
+For example:
++
+[source]
+----
+<property name="dashbuilder.runtime.location" value=" http://localhost:8080"
+----
+* You have set the `dashbuilder.export.dir` system property to the shared directory where Dashbuilder Runtime reads its model.
++
+For example:
++
+[source]
+----
+<property name="dashbuilder.export.dir" value="/tmp/dashbuilder/models/"
+----
 
 .Procedure
+
+. Start {CENTRAL}, go to *Menu → Design → Pages*.
+. In the *Components* panel, drag and edit the required component types to the canvas.
+. Click *Save*.
 . In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Dashbuilder Data Transfer*.
-. If you want to export all dashbuilder data as a ZIP file, click *Export all* on the *Dashbuilder Data Transfer* page.
-. If you want to select the data to be exported, click *Custom Export* on the *Dashbuilder Data Transfer* page.
+. On the *Dashbuilder Data Transfer* page, click *Custom export*.
+. In *Export Wizard* panel, verify the pages that you want to include in the ZIP file and click *Next*.
+. Select *Export Wizard* and click *Open*.
++
+You can see the dashbuilder runtime home page. If you are not logged in, you are redirected to the login page.
```

Comment:
Not sure, whether we need to capitalize dashbuilder runtime or not. Please confirm. For now, I am being consistent to what is already there. 
```suggestion
You can see the home page of the Dashbuilder Runtime. If you are not logged in, you are redirected to the login page.
```